bindings/node/package.json

```diff
@@ -7,20 +7,23 @@
     "lib": "lib"
   },
   "scripts": {
-    "install": "node-gyp rebuild",
+    "install": "prebuild-install || node-gyp rebuild",
     "format-cxx": "git-clang-format",
     "format-js": "prettier --print-width 100 --tab-width 2 --single-quote --write index.js 'test/**/*.js' 'lib/**/*.js'",
     "lint": "eslint lib test",
     "docs": "jsdoc2md --template etc/README.hbs --plugin dmd-clear --files lib/**/*.js > README.md",
-    "test": "mocha test"
+    "test": "mocha test",
+    "rebuild": "prebuild --compile",
+    "prebuild": "prebuild  --strip --verbose --tag-prefix node-v -t 10.16.0 -t 8.16.0 -t 6.17.1 -t 4.9.1"
```

Comment:
is there a reason we're not just building `--all`? Do we want to deprive users in e.g. electron, and do we want to get in to the habit of having to hand maintain this list of supported node versions?

Reply:
We are not build `--all` b/c Node 12 is broken right now. See https://jira.mongodb.org/browse/NODE-2097. It's the usual Nan stuff.